### ISSUE TITLE
Bootloader is no longer being written

### ISSUE THREAD

**Issue by trmazi** [pumpitupdev/pumpos#18]
Hello, I have not been able to write a booting pumpos image as the bootloader is not being written to the drive anymore. Following the readme files step by step, the end result is always 1 single partition. 

I am going to try a previous version here soon, and I'll add onto here if I find anything. I think that this has to do with the drive dependencies changing. During the install, the destination drive is no longer being typed as /dev/*drive*, but as *drive*. 

This may be causing issues during the bootloader install process.

**Comment by trmazi** [comment]
```
Setting up grub-common (2.02-2ubuntu8.23) ...
update-rc.d: warning: start and stop actions are no longer supported; falling back to defaults
Running in chroot, ignoring request.
```

Taken from my install log

**Comment by trmazi** [comment]
How to manually install grub
```
sudo mount --bind /dev /pumposmountpoint/dev
sudo mount --bind /proc /pumposmountpoint/proc
sudo mount --bind /sys /pumposmountpoint/sys
chroot /pumposmountpoint /bin/bash
grub-install --target=i386-pc /dev/sdX
```
where /dev/sdX is your pumpos disk

**Comment by voidderef** [comment]
Information exchange in live debug session allowed me to reproduce this on my Arch Linux host system. Using the provided mitigation, I was able to boot into the system using `./pumpos.sh os-qemu-boot`.

**Comment by voidderef** [comment]
Fix implemented, tested and confirmed working. Fix is included in latest 1.06 release.
